pkg/client/client_test.go

```diff
@@ -1,25 +1,32 @@
 package client
 
-import (
-	"testing"
-)
+// import (
+// 	"testing"
 
-var c = NewClient("/tmp/foo")
+// 	"github.com/weaveworks/gitops-toolkit/pkg/storage"
+// 	"github.com/weaveworks/gitops-toolkit/pkg/storage/cache"
 
-func TestClientGet(t *testing.T) {
-	vm, err := c.VMs().Get("1234")
-	t.Error(vm, err)
+// 	"github.com/weaveworks/ignite/pkg/apis/ignite/scheme"
+// )
 
-	t.Error(c.VMs().Delete("1234"))
+// var c = NewClient(cache.NewCache(
+// 	storage.NewGenericStorage(
+// 		storage.NewGenericRawStorage("/tmp/foo"), scheme.Serializer)))
 
-	vms, err := c.VMs().List()
-	t.Error(len(vms), err)
-}
```

Comment:
This and the following tests try to get a VM without creating the VM, which results in failure. Should this be removed for other tests that already use the client properly? Or should the test be fixed by creating a new VM object, querying and deleting it?

Reply:
Removed these tests. They seemed to be for development purpose only.